t/unit/backends/test_couchbase.py

```diff
@@ -106,15 +106,3 @@ def test_backend_params_by_url(self):
             assert x.username == 'johndoe'
             assert x.password == 'mysecret'
             assert x.port == 123
-
```

Comment:
just curious why the test is being dropped? or am I missing something?

Reply:
The method is not being used by the couchbase results backend at all.
Also, it no longer behaves as expected.
A future refactor of that method should ensure that if we're converting every key to string we'll also decode byte strings to UTF-8.